src/js/components/Text/doc.js

```diff
@@ -74,6 +74,11 @@ export const doc = Text => {
         'large',
         'xlarge',
         'xxlarge',
+        '2xl',
```

Comment:
Line 76 seems like a duplicate of 77.

Reply:
76 is backwards compatible, 77 is friendlier for the new naming, both seem reasonable to me

Reply:
seeing them both as consecutive sizes gives the impression that '2xl', is larger than 'xxlarge'.

Reply:
I believe it's somewhat standard in clothing sizes that "2XL" and "XXLARGE" mean the same thing. We can add a note in the documentation that they are identical.